schemas/skills/readme.md

```diff
@@ -0,0 +1,11 @@
+# Skills JSON schema
+
+This folder contains the current and previous versions of the JSON Schema that defines your skill manifest. For additional information on skills, skill consumers, and skill manifests see the [skills documentation](https://docs.microsoft.com/azure/bot-service/skills-conceptual?view=azure-bot-service-4.0).
```

Comment:
Consider creating an aka.ms alias here in case we move the docs so we don't need to do a new PR to update this. 
@Kaiqb, knows the alias of the team that needs to be added to the aka.ms entry to tech writers and see it and update it if needed.